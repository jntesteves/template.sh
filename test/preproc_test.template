%{{{{ printf foo}}}}
{{{printf foo}}}
{{{ printf foo }}}
{{{     printf foo    }}}
{{{ echo "foo" }}
{{{ printf 'foo' }}}
------------------------------------------
{{{ printf "f"o"o"; printf 'b'a'r' }}}
{{{ echo foo {{{ printf bar }}} echo foo }}}
------------------------------------------
{{{{ }}}
{{{ }}} }}}
{{{ {{{ }}}
{{{}}}}}}
{{{ {{{ }}}
{{{ }}}{{{ }}}
------------------------------------------
{{{
    {{{
        {{{
            {{{ }}}
        }}}
    }}}

}}}
------------------------------------------
{{{echo foo;printf bar}}}
------------------------------------------
{{{
    printf bar
}}}
