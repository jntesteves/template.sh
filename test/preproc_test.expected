%{foo}
foo
foo
foo
{{{ echo "foo" }}
foo
------------------------------------------
foobar
{{{ echo foo bar echo foo }}}
------------------------------------------
{
 }}}
{{{ 
}}}
{{{ 

------------------------------------------
{{{
    {{{
        {{{
            
        }}}
    }}}

}}}
------------------------------------------
foo
bar
------------------------------------------
bar
